bundle/manifests/confidentialcontainers.org_ccruntimes.yaml

```diff
@@ -5686,17 +5686,22 @@ spec:
                     description: This specifies the RuntimeClasses that need to be
                       created, with its name and an associated snapshotter to be used
                     items:
-                      description: RuntimeClass holds the name and the snapshotter
```

Comment:
I'm not exactly sure how this works, but re-running this on your PR I'm getting many more changes to this file. I tried doing it fresh on the main branch and that also brought many more changes to this manifest. Is this expected, did some dependency changed in the mean-time?

Reply:
@wainersm could you please elaborate a bit on this? I'm getting completely different results by running the `make bundle IMG=quay.io/confidential-containers/operator:v0.9.0`